backend/src/main/java/gov/cdc/usds/simplereport/api/DeviceMigrationController.java

```diff
@@ -0,0 +1,17 @@
+package gov.cdc.usds.simplereport.api;
+
+import gov.cdc.usds.simplereport.service.DeviceMigrationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
```

Comment:
fyi I had to make this a `@RestController` annotation to avoid getting some strange "template parsing" error from Thymeleaf.